src/layers/legacy/medCoreLegacy/database/medLocalDbController.cpp

```diff
@@ -107,16 +107,30 @@ bool medLocalDbController::createStudyTable()
 {
     QSqlQuery query(m_database);
 
-    return query.prepare(
-               "CREATE TABLE IF NOT EXISTS study ("
-               " id        INTEGER      PRIMARY KEY,"
-               " patient   INTEGER," // FOREIGN KEY
-               " name         TEXT,"
-               " uid          TEXT,"
-               " thumbnail    TEXT,"
-               " studyId      TEXT"
-               ");") &&
+    query.prepare(
+        "CREATE TABLE IF NOT EXISTS study ("
+        " id        INTEGER      PRIMARY KEY,"
+        " patient   INTEGER," // FOREIGN KEY
+        " name         TEXT,"
+        " uid          TEXT,"
+        " thumbnail    TEXT,"
+        " studyId      TEXT,"
+        " time         TEXT,"
+        " date         TEXT"
+        ");") &&
            execQuery(query, __FILE__, __LINE__);
```

Comment:
it's also done line 125, is it normal?

Reply:
I want the same comportement as createSeriesTable() so I copied the code from that function

Reply:
I want the same comportement as createSeriesTable() so I copied the code from that function